app/services/users.js

```diff
@@ -16,3 +16,20 @@ exports.saveUser = async data => {
     throw databaseError(err.errors);
   }
 };
+
+exports.verifyCredentials = async (email, password) => {
+  try {
+    const user = await db.User.findOne({ where: { email } });
```

Comment:
I think that here there are two logics together and they could be separated, don't you think it would be better to have a method to verify if the user exists and another to verify the user's password?

Reply:
In this case I may created one function for get the user and other for verify email and password?

Reply:
of course it would be much better this way